### DISCUSSION TITLE
Using Yarn Create

### DISCUSSION THREAD

**Discussion by fweth** [codesandbox/codesandbox-client#6644] [General] [answered]
What's the way to go when I want to try out some framework for which no CSB starter is available?  I'd like to do `yarn create some-framework-app .` in the sandbox folder (in a fresh Node.js project).  But there is a `.codesandbox` folder, which I guess is important, so usually I move that folder to `/tmp`, then delete everything in the sandbox folder, perform `yarn create some-framework-app .` and move `.codesanbox` back afterwards.  Is there a better / more elegant way to do this?

**Comment by JamesACS** [comment, marked as answer]
Hey @fweth - We have a vanilla template, which is probably our most basic! 

I'm afraid I don't know of a more elegant method beyond what you've put here
